fastlane/lib/fastlane/actions/ensure_git_status_clean.rb

```diff
@@ -23,7 +23,25 @@ def self.run(params)
         # Manual post processing trying to ignore certain file paths
         if (ignore_files = params[:ignore_files])
           repo_status = repo_status.lines.reject do |line|
-            path = line.split(' ').last
+            # Extract the file path from the line based on
+            # https://git-scm.com/docs/git-status#_output
+
+            # The first two characters indicate the status of the file path (e.g. ' M')
+            # XY PATH
+            path = line&.match(/^.{2}\s(.*)/)&.[](1) || ''
```

Comment:
Why are we using `&` here? How can `line` be `nil`?

Reply:
Good catch, indeed, a line cannot be `nil`. Fixed.